### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ctzsnooze**
Include advice about categories

### CODE REVIEW

**Review by limonspb — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -24,19 +24,72 @@ Configuration Snippets for the Betaflight Flight Controller Firmware
...
+If a deadband is appied, ensure this is noted in the description.  
+Throttle curves should not be included in a Rates preset.  Throttle curves should not be included in `RATES` presets.   Combinations of throttle curve presets (perhaps including throttle boost or thrust linear, for example) may be included with `RATES` may shared as a `MIXED` type of preset.
+
+## OSD
+
+OSD configurations including layout, elements, etc
+
+## VTx
+
+VTx configurations including channels and CLI entries.  Can be general, or specific to certain hardware, or boards.
```
[the remark this answers is not in the document]
Comment (ctzsnooze) [reply README.md:50]: Betaflight has had a 'policy' about not providing VTx tables formally in the code, making it the user's responsibility to add them in a way that is consistent with the legal requirements of the country.  We have to check how that would apply here - specifically, whether or not we can provide presets here, which require user action to apply, and, if so, what kind of disclaimer would be required.

File: README.md [hunk 2]
```diff
@@ -24,19 +24,72 @@ Configuration Snippets for the Betaflight Flight Controller Firmware
...
+
+PIDs and related values such as Dmin, Dmin gain, antigravity, throttle boost, etc
+Please summarise the altered parameter groups in a description field.
+Always explain the intent of the tune, and the kind of build to which it applies, including motors, props, the lot, using the description fields
+
+## Rates
+
+RC Rates and deadbands.  
+If a deadband is appied, ensure this is noted in the description.  
+Throttle curves should not be included in a Rates preset.  Throttle curves should not be included in `RATES` presets.   Combinations of throttle curve presets (perhaps including throttle boost or thrust linear, for example) may be included with `RATES` may shared as a `MIXED` type of preset.
```
[the remark this answers is not in the document]
Comment (ctzsnooze) [reply README.md:42]: modified and i hope improved the text

File: README.md [hunk 3]
```diff
@@ -24,19 +24,72 @@ Configuration Snippets for the Betaflight Flight Controller Firmware
...
 
 ### Official
 ```
     True, False
 ```
 
 ### include
-Optional paths to snippet files to be included. Works similar to C/C++ `#include`.
-Allows multiple files to be included. Recursion is not supported yet. Metaproperties of the included files are being ignored.
+Optional path to other preset files, to inset their contents within the current preset data, ad the point where the  `include` occurs.
```
[the remark this answers is not in the document]
Comment (ctzsnooze) [reply README.md:89]: fixed

**Comment by ctzsnooze** [comment]
Please note that the Read Me document is an active work in progress.  Any comments and feedback are very welcome.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (limonspb) [reply README.md:50]: I was thinking it's a policy for the firmware. Should be fine with the disclaimer for each vtx Table preset. @mikeller  what you think?

**Pull request closed without merging**